extensions/amp-sticky-ad/0.1/amp-sticky-ad.js

```diff
@@ -61,6 +64,10 @@ class AmpStickyAd extends AMP.BaseElement {
         'amp-sticky-ad must have a single amp-ad child');
 
     this.ad_ = children[0];
+    const adType = this.element.getAttribute('type');
+    const config = adConfig[this.ad_.getAttribute('type')];
```

Comment:
I would like to do that! but then I would need to access the implemention_ for the amp-ad element. I don't think that's a good a approach and adding the method to `custom-element` doesn't sound good to me either. Any idea?